### ISSUE TITLE
Fix System.IO.FileNotFoundException: Could not load file or assembly 'Accessibility, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified. in CulturePreservingExecutionContext.CallbackWrapper (Object obj)

### ISSUE THREAD

**Issue by millerscout** [millerscout/Kenshi-Mod-Manager#65]
### Stacktrace ###

MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(Object obj);MS.Internal

MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state);MS.Internal

MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled);MS.Win32

MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o);MS.Win32

MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam);MS.Win32

MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg);MS.Win32

KenshiModTool.App.Main();KenshiModTool

### Reason ###

System.IO.FileNotFoundException: Could not load file or assembly 'Accessibility, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified.

### Link to App Center ###

* [https://appcenter.ms/users/mmillerdantas/apps/kenshi-mod-manager/crashes/errors/1211084385u](https://appcenter.ms/users/mmillerdantas/apps/kenshi-mod-manager/crashes/errors/1211084385u)
